tests/robot-tests/scripts/permalink_snapshots/checked-permalinks.csv

```diff
@@ -0,0 +1 @@
+permalink_id, response_time
```

Comment:
Do we want to .gitignore this file and also the resulting csvs from prepping and running the script?  Just not wanting anyone to accidentally check in all of the Permalink IDs from Prod into the codebase!

If we do this, I guess the script will need adjusting slightly just to add manual instructions to create this CSV, in the same way there are manual instructions to create `invalid-permalinks.csv`. 

Either that or we just make sure no-one checks in any files, but my preference would be the .gitignore route.

Reply:
Yep, that sounds like a good idea. Wasn't sure on whether to exclude it or not